deployment-template/ec2/data-prepper-ec2-deployment-cfn.yaml

```diff
@@ -191,7 +191,14 @@ Resources:
           tar -xzf /tmp/$RELEASE.tar.gz --directory /usr/local/bin
           /opt/aws/bin/cfn-init -v --stack ${AWS::StackId} --resource EC2Instance --region ${AWS::Region} --configsets default
           nohup /usr/local/bin/$RELEASE/data-prepper-tar-install.sh /etc/data-prepper/pipelines.yaml /etc/data-prepper/data-prepper-config.yaml > /var/log/data-prepper.out &
-          /opt/aws/bin/cfn-signal --stack ${AWS::StackId} --resource EC2Instance --region ${AWS::Region}
+          sleep 5s
+          export EXCEPTION_MSG=$(grep -E "Exception" /var/log/data-prepper.out)
```

Comment:
There's a chance of the actual log file not existing (ex: if we type in a version of Data Prepper that doesn't exist, the tar.gz will fail to download). I imagine this grep alone won't catch that.

Reply:
No. It won't. Checking ways of enhancement.